contrib/hf-samples/2-node-network.groovy

```diff
@@ -0,0 +1,39 @@
+//! Simulation
+
+import org.arl.fjage.RealTimePlatform
+import org.arl.unet.sim.*
+import org.arl.unet.sim.channels.*
+
+///////////////////////////////////////////////////////////////////////////////
+// display documentation
+
+println '''
+2-node network (High-frequency)
+-------------------------------
+
+Node A: tcp://localhost:1101, http://localhost:8081/
+Node B: tcp://localhost:1102, http://localhost:8082/
+'''
+
+///////////////////////////////////////////////////////////////////////////////
+// simulation settings
+
+platform = RealTimePlatform           // use real-time mode
+
+///////////////////////////////////////////////////////////////////////////////
+// channel and modem settings
+
+channel = [
+  model:                BasicAcousticChannel,
+  carrierFrequency:     60.kHz,
+  bandwidth:            20000.Hz,
+  waterDepth:           30.m
```

Comment:
water depth 10 - 20m?

Reply:
Got this information on depth from the proposal here : https://github.com/subnero1/project-management/blob/master/projects/2019/%5BWNC%5D%20Baby%20Raptor%20%5BSG%5D/Prior/High-speed%20modem%20Proposal.md

It mentions 1 m - 30 m.